### ISSUE TITLE
table of elevation has chunk based detection

### ISSUE THREAD

**Issue by GanerCodes** [barribob/bosses-of-mass-destruction#131]
I have the radius set to max and noticed my flight drops out lopsided compared to the table. Turns out this happens at a chunk border. After staring at code for a sec i foujnd the issue
![image](https://github.com/user-attachments/assets/0a0d4627-0a5a-4695-ad61-d66a0b167973)

basically you're only checking the border and center, so 32 radius is 2 chubnks away

idk kotlin but the solution is prob would look like 
```kotlin
val r = tableOfElevationRadius.toInt();
val j = r / 16 + 1;
for (x in -j..j) {
    for (z in -j..j) {
        blockToCheck.add(BlockPos(x * r, 0, z * r))
    }
}
```

**Comment by chudders1231** [comment]
Is this still being maintained / is this still being worked on as i can see this is an issue present in Prominent 2: RPG

**Comment by barribob** [comment]
Should be fixed in 1.10.2
